apps/frontend/src/App.tsx

```diff
@@ -10,6 +19,7 @@ import {
 } from "./components";
 import { lightTheme } from "./theme";
 import { GoogleReCaptchaProvider } from "react-google-recaptcha-v3";
+import { Disclaimer } from "./components/Disclaimer/Disclaimer.tsx";
```

Comment:
should import from `./components`

Reply:
Done here https://github.com/vechain/x-app-template/pull/12/commits/467c9510d0d480a83abf34fdc201ce3f745a705c